Helpers/RLMRealm+Notifications.m

```diff
@@ -65,4 +65,26 @@ - (void)deleteObjectsWithNotification:(id<NSFastEnumeration>)array
     [self deleteObjects:array];
 }
 
+#pragma mark - private helpers
+
+- (BOOL)rbq_containsObject:(RLMObject *)object {
+    if (!object) {
+        return NO;
+    }
+    
+    Class objectClass = [object class];
+    
+    RLMProperty *primaryKeyProperty = object.objectSchema.primaryKeyProperty;
+    
+    if (primaryKeyProperty) {
+        id primaryKeyValue = [object objectForKeyedSubscript:[objectClass primaryKey]];
+        
+        if (!primaryKeyValue) {
+            return NO;
+        }
+        
+        return !![[object class] objectInRealm:self forPrimaryKey:primaryKeyValue];
```

Comment:
This should then use `NSClassFromString()` to use the class name and get the appropriate class.